### PR TITLE
Updates notifications using native module

### PR DESCRIPTION
**Pull request by sarahscott**
Adds a way to replace [this](https://github.com/artsy/eigen/blob/master/Artsy/View_Controllers/WorksForYou/ARWorksForYouViewController.m#L170)

### CODE REVIEW

File: src/lib/containers/works_for_you.tsx [hunk 1]
```diff
@@ -32,6 +36,19 @@ export class WorksForYou extends React.Component<Props, State> {
 
   }
 
+  componentDidMount() {
+    NativeModules.ARTemporaryAPIModule.markNotificationsRead((error) => {
+      if (error) {
+        console.error(error)
+      } else {
+        Events.postEvent(this, {
```
Comment (sarahscott) [inline src/lib/containers/works_for_you.tsx]: optional analytics ✨

**Comment by alloy** [comment]
This looks great 👍 

However, the ‘TemporaryAPIModule’ is only meant for networking, i.e. ‘API’ refers to Gravity. It is meant to be temporary in the sense that it would get removed once we have mutations in MP.

So can you add a dedicated native module for it instead? E.g. ‘ARWorksForYouModule’

**Comment by sarahscott** [comment]
Ah yeah that  makes sense. Will do. How temporary is the `ARTemporaryAPIModule`? Are there plans to make metaphysics mutable soon?

**Comment by alloy** [comment]
No planned timeframe, just whenever we get to it.

**Comment by sarahscott** [comment]
Alright, 🍏 and ready for review!

**Comment by alloy** [comment]
Looks good! 👌

**Pull request merged**